### PR TITLE
Added Documentation for /filter endpoints

### PR DESCRIPTION
**Pull request by gravishpurohit**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: handlers/server_spec_handler.go [hunk 1]
```diff
@@ -19,6 +19,13 @@ type Version struct {
 	ReleaseChannel string `json:"release_channel,omitempty"`
 }
 
+// swagger :route GET /api/server/version SystemAPI idGetSystemVersion
+// Handle GET reqest for system/server version
+//
+//Returns the running Meshery version
+// responses:
+//200: mesheryVersionRespWrapper
```
Comment (leecalcote) [inline handlers/server_spec_handler.go:27]: @piyushsingariya @navendu-pottekkat @dhruv0000 are we not giving example responses in docs?

this description doesn’t really tell me the purpose of the endpoint and it’s intended use.

Comment (piyushsingariya) [reply handlers/server_spec_handler.go:28]: we do have samples included in docs. I am not sure why these swagger comments have been edited! @gravishpurohit

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for jumping in and engaging in this effort, @gravishpurohit. It's great to have you engaged.

Note that your last commit added many other commits from other contributors. You'll want to resync your fork and clone with the upstream repo in order to only include your commits in this PR.

**Comment by gravishpurohit** [comment]
alright got it, trying again.
 thanks @leecalcote!

**Comment by leecalcote** [comment]
Good deal, @gravishpurohit. Before you make another attempt, though, please address @piyushsingariya's question.

File: handlers/server_spec_handler.go (hunk 1 above)
Comment (gravishpurohit) [reply handlers/server_spec_handler.go:28]: @piyushsingariya ya, as per the docs I added that and I didn't edit.

Comment (piyushsingariya) [reply handlers/server_spec_handler.go:28]: reset this file `handlers/server_spec_handler.go`, you have edited this, not sure why. This doesn't even have anything related to `/filter` endpoints.

**Comment by leecalcote** [comment]
heads-up: lint failure.

**Comment by leecalcote** [comment]
Hi @iamsdas, do you think you can finish off this PR? @gravishpurohit seems to be unavailable.

**Comment by iamsdas** [comment]
@leecalcote sure, I'll make a new PR

**Comment by navendu-pottekkat** [comment]
Closing this PR since @iamsdas will make a new PR

**Pull request closed without merging**